config

```diff
@@ -21,7 +21,8 @@ keeper.keep_warm true
 # Number of active decks
 keeper.decks 2
 
-
+# == sACN ==
+sacn.enabled true
```

Comment:
I would like all config options to already be added in this file with their default values. Makes it easier for user to discover the options available